DANS-KNAW/easy-convert-bag-to-deposit
# DD-858: keep valid non-zero/empty spatials

Fixes DD-858: keep spatials

When applied it will
--------------------
* 
* 
* 

Where should the reviewer @DANS-KNAW/easy start?
------------------------------------------------

How should this be manually tested?
-----------------------------------
* [x] ran `mvn clean install` witout `-DskipTests `
* [x] download the bag attached to the issue and execute

      run.sh -u data/DD-858/5e766f95-3ec2-4b51-a0cb-5ae874771704  -t DOI -s FEDORA
   
  Compare the changes in the `dataset.xml`. Significant changes should not involve spatal elements.

Related pull requests on github
-------------------------------

repo                       | PR                | note
-------------------------- | ----------------- | ----
easy-                      | [PR#](PRlink)     | ..
